1-exercises/A-accessing-values/exercise3.js

```diff
@@ -20,6 +20,15 @@ let basketballTeam = {
     - console.logs the name of each player on a new line
 */
 
+console.log(basketballTeam.topPlayers);
+
+let sortedTopPlayer = basketballTeam.topPlayers.sort();
+console.log(sortedTopPlayer);
```

Comment:
make sure you don't leave extra console logs in by accident, could cause issues with tests if you do it on the wrong function